### ISSUE TITLE
Special Characters in Album/Artist name causing Crash

### ISSUE THREAD

**Issue by charliekempf** [torch2424/live-stream-radio#157]
Here is my error log

`ffmpeg stderr:

 ffmpeg version 4.1.6-1~deb10u1 Copyright (c) 2000-2020 the FFmpeg developers
  built with gcc 8 (Debian 8.3.0-6)
  configuration: --prefix=/usr --extra-version='1~deb10u1' --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --arch=amd64 --enable-gpl --disable-stripping --enable-avresample --disable-filter=resample --enable-avisynth --enable-gnutls --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libcodec2 --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libjack --enable-libmp3lame --enable-libmysofa --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librsvg --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzmq --enable-libzvbi --enable-lv2 --enable-omx --enable-openal --enable-opengl --enable-sdl2 --enable-libdc1394 --enable-libdrm --enable-libiec61883 --enable-chromaprint --enable-frei0r --enable-libx264 --enable-shared
  libavutil      56. 22.100 / 56. 22.100
  libavcodec     58. 35.100 / 58. 35.100
  libavformat    58. 20.100 / 58. 20.100
  libavdevice    58.  5.100 / 58.  5.100
  libavfilter     7. 40.101 /  7. 40.101
  libavresample   4.  0.  0 /  4.  0.  0
  libswscale      5.  3.100 /  5.  3.100
  libswresample   3.  3.100 /  3.  3.100
  libpostproc    55.  3.100 / 55.  3.100
Input #0, matroska,webm, from '/home/charlie/Siivagunner2/video/Sequence 01.webm':
  Metadata:
    encoder         : libwebm-0.2.1.0
    creation_time   : 2020-12-05T07:11:51.000000Z
  Duration: 00:02:00.12, start: -0.007000, bitrate: 855 kb/s
    Stream #0:0(eng): Video: vp9 (Profile 0), yuv420p(tv), 720x480, SAR 1:1 DAR 3:2, 30 fps, 30 tbr, 1k tbn, 1k tbc (default)
    Stream #0:1(eng): Audio: opus, 48000 Hz, stereo, fltp (default)
Input #1, mp3, from '/home/charlie/Siivagunner2/audio/Battle Win - Sonic Chronicles - The Dark Brotherhood.mp3':
  Metadata:
    major_brand     : isom
    minor_version   : 512
    compatible_brands: isomiso2mp41
    purl            : https://www.youtube.com/watch?v=QUzqZkuJxnw
    comment         : Music: Battle Win
                    : Composer: Steven Sim
                    : Playlist: https://www.youtube.com/playlist?list=PLL0CQjrcN8D37VeNi2qu2i29QY6OVNTLo
                    : Platform: Nintendo DS
                    :
                    : Please read the channel description.
    album           : Sonic Chronicles: The Dark Brotherhood
    description     : Music: Battle Win
                    : Composer: Steven Sim
                    : Playlist: https://www.youtube.com/playlist?list=PLL0CQjrcN8D37VeNi2qu2i29QY6OVNTLo
                    : Platform: Nintendo DS
                    :
                    : Please read the channel description.
    artist          : Battle Win
    title           : Battle Win - Sonic Chronicles: The Dark Brotherhood
    date            : 20190910
    encoder         : Lavf58.20.100
  Duration: 00:00:08.10, start: 0.025057, bitrate: 321 kb/s
    Stream #1:0: Audio: mp3, 44100 Hz, stereo, fltp, 320 kb/s
    Metadata:
      encoder         : Lavc58.35
Input #2, lavfi, from 'anullsrc':
  Duration: N/A, start: 0.000000, bitrate: 705 kb/s
    Stream #2:0: Audio: pcm_u8, 44100 Hz, stereo, u8, 705 kb/s
[Parsed_drawtext_6 @ 0x55600f396d80] Both text and text file provided. Please provide only one
[AVFilterGraph @ 0x55600f378f00] Error initializing filter 'drawtext' with args 'text=Album: Sonic Chronicles: The Dark Brotherhood:fontfile=/home/charlie/Siivagunner2/./fonts/Lato-Regular.ttf:fontsize=(w * 0.023333333333333334):bordercolor=#000000:borderw=1:fontcolor=#FFFFFF:y=(h * 0.25):x=(w * 0.02)'
Error initializing complex filters.
Invalid argument

ffmpeg stdout:

ffmpeg err:

 Error: ffmpeg exited with code 1: Error initializing complex filters.
Invalid argument

    at ChildProcess.<anonymous> (/home/charlie/.nvm/versions/node/v15.3.0/lib/node_modules/live-stream-radio/node_modules/fluent-ffmpeg/lib/processor.js:182:22)
    at ChildProcess.emit (node:events:376:20)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:277:12)

ffmpeg encountered an error. 😨
Please see the stderror output above to fix the issue.
`

I think the reason it crashed was because of the colon in the title "Sonic Chronicles: The Dark Brotherhood," I would try to fix it myself but I have very little coding experience

**Comment by AndreasWebdev** [comment]
Yes, this is very likely. I've tried fixing this back then but failed doing so. For now, you could strip out colons in your MP3 tags.
